### PR TITLE
Issue with dual screen on Live FlashPlayer 

### PR DESCRIPTION
**Pull request by shalommeoded**
@eitanavgil  Please review this PR

### CODE REVIEW

File: modules/EmbedPlayer/resources/mw.EmbedPlayerKplayer.js [hunk 1]
```diff
@@ -979,7 +979,7 @@
 			if (source !== -1) {
 				sourceIndex = this.getSourceIndex(source);
 			}
-			this.playerObject.sendNotification('doSwitch', {flavorIndex: sourceIndex});
+			this.playerObject.sendNotification('doSwitch', {flavorIndex: this.getSourceIndex(source)});
```
[the remark this answers is not in the document]
Comment (eitanavgil) [reply modules/EmbedPlayer/resources/mw.EmbedPlayerKplayer.js]: https://github.com/kaltura/chromeless-kdp/blob/4660bb8aa483cc16e37100ee564253de51f398c5/lightKdp3Lib/src/com/kaltura/kdpfl/view/media/KMediaPlayerMediator.as#L435

Seems that passing -1 is to tell the player to turn on Auto Switch mode.

[the remark this answers is not in the document]
Comment (OrenMe) [reply modules/EmbedPlayer/resources/mw.EmbedPlayerKplayer.js]: @eitanavgil yes, so this is valid and we should leave the original logic - as it passes signal to flash to set `auto`.
What is the value that is passed to the flash player in your use case? what is `source` object being passed to `getSourceIndex` and what value does it return?

[the remark this answers is not in the document]
Comment (eitanavgil) [reply modules/EmbedPlayer/resources/mw.EmbedPlayerKplayer.js]: it passes -1 today and this is what is breaking it.

[the remark this answers is not in the document]
Comment (OrenMe) [reply modules/EmbedPlayer/resources/mw.EmbedPlayerKplayer.js]: Yes, but passing null or undefined is not a supported value, so -1 is valid and you need to understand why the specific stream breaks.
I do not know even if the error use case is when -1 is passed, is this the issue?

**Comment by OrenMe** [comment]
@shalommeoded what is the status of this fix? Please update.

**Comment by OrenMe** [comment]
closing due to inactivity, please reopen to latest if still relevant

**Pull request closed without merging**